kge/config-default.yaml

```diff
@@ -492,22 +492,28 @@ entity_ranking:
   # mean_reciprocal_rank_filtered_with_test.
   filter_with_test: True
 
-  # How to handle cases with ties between the correct answer and other answers, e.g.,
-  #  Query: (s, p, ?).
-  #  Answers and score: a:10, b:10, c:10, d:11, e:9
-  #  Correct: 'a'.
-  #
-  # Possible options are:
-  # - worst_rank:        Use the highest rank of all answers that have the same
-  #                      score as the correct answer. In example: 4.
-  # - best_rank:         Use the lowest rank of all answers that have the same
-  #                      score as the correct answer (competition scoring). In
-  #                      example: 2.
-  #                      DO NOT USE THIS OPTION, it leads to misleading evaluation
-  #                      results. See https://arxiv.org/pdf/1911.03903.pdf
-  # - rounded_mean_rank: Average between worst and best rank, rounded up
-  #                      (rounded fractional ranking). In example: 3.
-  tie_handling: rounded_mean_rank
+  tie_handling:
+
+    # How to handle cases with ties between the correct answer and other answers, e.g.,
+    #  Query: (s, p, ?).
+    #  Answers and score: a:10, b:10, c:10, d:11, e:9
+    #  Correct: 'a'.
+    #
+    # Possible options are:
+    # - worst_rank:        Use the highest rank of all answers that have the same
+    #                      score as the correct answer. In example: 4.
+    # - best_rank:         Use the lowest rank of all answers that have the same
+    #                      score as the correct answer (competition scoring). In
+    #                      example: 2.
+    #                      DO NOT USE THIS OPTION, it leads to misleading evaluation
+    #                      results. See https://arxiv.org/pdf/1911.03903.pdf
+    # - rounded_mean_rank: Average between worst and best rank, rounded up
+    #                      (rounded fractional ranking). In example: 3.
+    type: rounded_mean_rank
+
+    # Tie detection tolerances i.e. how close should scores be to count them as ties, see torch.isclose for more details.
```

Comment:
Apply formatting. I suggest to write a bit more here to be clear. E.g., "# Tie detection tolerances. Two scores count as a tie whenever they fall within this tolerance; see torch.isclose for details. Important to increase numerical stability (see <link-to-bug-report>."